homeassistant/components/permobil/coordinator.py

```diff
@@ -0,0 +1,59 @@
+"""DataUpdateCoordinator for permobil integration."""
+
+import asyncio
+from dataclasses import dataclass
+from datetime import timedelta
+import logging
+
+from mypermobil import (
+    ENDPOINT_BATTERY_INFO,
+    ENDPOINT_DAILY_USAGE,
+    ENDPOINT_VA_USAGE_RECORDS,
+    MyPermobil,
+    MyPermobilAPIException,
+)
+
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@dataclass
+class MyPermobilData:
+    """MyPermobil data stored in the DataUpdateCoordinator."""
+
+    battery: dict[str, str | float | int | list | dict]
+    daily_usage: dict[str, str | float | int | list | dict]
+    records: dict[str, str | float | int | list | dict]
+
+
+class MyPermobilCoordinator(DataUpdateCoordinator[MyPermobilData]):
+    """MyPermobil coordinator."""
+
+    def __init__(self, hass: HomeAssistant, p_api: MyPermobil) -> None:
+        """Initialize my coordinator."""
+        super().__init__(
+            hass,
+            _LOGGER,
+            name="permobil",
+            update_interval=timedelta(seconds=50),
```

Comment:
You set the interval here to 50 seconds, but your third-party library is caching the calls up to 5 minutes.

```suggestion
            update_interval=timedelta(minutes=5),
```